### DISCUSSION TITLE
Atmosperes PLEASE!!

### DISCUSSION THREAD

**Discussion by ZaRaTuStRa666** [willwulfken/MidJourney-Styles-and-Keywords-Reference#44] [Ideas and Style Submissions]
Atmospheres in a prompt compose the image in styles and/or perspectives that derive specific human emotions, e.g. "dark atmosphere" will elicit fear/danger, better example, if one prompts: "A gothic alien cathedral by Giger" you would likely get an awesome image but adding "entering atmosphere" at the end of the prompt the image will shift the perspective to evoke the expereince of awe and mysterious wonder of arriving at a place or structure you've never been to before.

**Comment by willwulfken** [comment]
I like this. I'll have to try this out. Thank you for the suggestion
